crates/common/mqtt_channel/src/config.rs

```diff
@@ -40,6 +42,11 @@ pub struct Config {
     ///
     /// Default: `1024 * 1024`.
     pub max_packet_size: usize,
+
+    /// Maximum time the connection will be alive
```

Comment:
I hope the connection can survive longer ;-)

```suggestion
    /// Maximum time between two heart beats for the connection to be kept alive
```

Reply:
fixed